### PR TITLE
Bump cpp-peglib/1.3.1

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **cpp-peglib/1.3.1**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
I've tested, and it fails at runtime with all gcc versions on Linux :(

File: recipes/cpp-peglib/1.x.x/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        self.copy("peglib.h", dst="include", src=self._source_subfolder)
+
+    def package_info(self):
+        if self.settings.os == "Linux":
+            self.cpp_info.system_libs = ["pthread"]
+            self.cpp_info.cxxflags.append("-pthread")
+            self.cpp_info.exelinkflags.append("-pthread")
+            self.cpp_info.sharedlinkflags.append("-pthread")
+        if self.settings.compiler == "Visual Studio":
+            self.cpp_info.cxxflags.append("/Zc:__cplusplus")
```
Comment (prince-chrismc) [inline recipes/cpp-peglib/1.x.x/conanfile.py:67]: https://docs.microsoft.com/en-us/cpp/build/reference/zc-cplusplus?view=msvc-160 Hmmm

**Review by prince-chrismc — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

File: recipes/cpp-peglib/1.x.x/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            min_length = min(len(lv1), len(lv2))
+            return lv1[:min_length] < lv2[:min_length]
+
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if not minimum_version:
+            self.output.warn("{} {} requires C++17. Your compiler is unknown. Assuming it supports C++17.".format(self.name, self.version))
+        elif lazy_lt_semver(str(self.settings.compiler.version), minimum_version):
+            raise ConanInvalidConfiguration("{} {} requires C++17, which your compiler does not support.".format(self.name, self.version))
+
+        if self.settings.compiler == "clang" and tools.Version(self.settings.compiler.version) == "7" and \
```
Comment (theirix) [inline recipes/cpp-peglib/1.x.x/conanfile.py:45]: _compilers_minimum_version declares minimal version 6 for clang. Should we set stdcpp for clang 6 too?

Comment (SpaceIm) [reply recipes/cpp-peglib/1.x.x/conanfile.py:45]: No, cpp-peglib works fine with clang 6 and libstdc++. I guess it's a bug in clang 7.

**Review by theirix — APPROVED** [review]

**Pull request merged**